platform/platform_linux.cpp

```diff
@@ -33,8 +34,8 @@
 
 namespace
 {
-// Web service ip to check internet connection. Now it's a GitHub.com IP.
-char constexpr kSomeWorkingWebServer[] = "140.82.121.4";
+// Web service ip to check internet connection. Now it's cdn.organicmaps.app IP.
+char constexpr kSomeWorkingWebServer[] = DEFAULT_CONNECTION_CHECK_IP;
```

Comment:
Can ip octets be used directly instead of a string? :)

Reply:
Changed.